pystdf/script/count.py

```diff
@@ -19,59 +19,51 @@
 #
 
 from __future__ import print_function
-import sys, os, re
+import sys
+import re
 
 try:
     import gzip
-    have_gzip = True
 except ImportError:
-    have_gzip = False
+    gzip = None
 try:
     import bz2
-    have_bz2 = True
 except ImportError:
-    have_bz2 = False
+    bz2 = None
 
 from pystdf.IO import Parser
 from pystdf.Indexing import RecordIndexer
-import pystdf.V4
 
-#def info(type, value, tb):
-#    import traceback, pdb
-#    # You are not in interactive mode; print the exception
-#    traceback.print_exception(type, value, tb)
-#    print
-#    # ... then star the debugger in post-mortem mode
-#    pdb.pm()
-#sys.excepthook = info
 
-gzPattern = re.compile('\.g?z', re.I)
-bz2Pattern = re.compile('\.bz2', re.I)
+GZ_PATTERN = re.compile('\.g?z', re.I)
+BZ2_PATTERN = re.compile('\.bz2', re.I)
```

Comment:
```suggestion
GZ_PATTERN = re.compile('\.g?z$', re.I)
BZ2_PATTERN = re.compile('\.bz2$', re.I)
```

These are better names, just noticing I also forgot to anchor the file extension. Total driveby.

It might also be reasonable to support stdin in the CLI, so one could pipe uncompressed output directly, without relying on this tool to do everything.

Reply:
Looks like this auto-uncompressing logic is repeated several places. We might refactor that...